FirebaseRemoteConfig/Sources/RCNConfigFetch.m

```diff
@@ -585,9 +584,10 @@ - (NSURLSessionDataTask *)URLSessionDataTaskWithContent:(NSData *)content
 }
 
 - (NSString *)getTemplateVersionNumber:(NSDictionary *)fetchedConfig {
```

Comment:
here it looks like we are still trying to read from the actual config.. however it doesn't look like we need this method anymore?

Reply:
The template version variable is stored in memory in this file. It would be initiated from disk using RCNConfigSettings and then updated every fetch request as well as saved to disk. This function is used to extract it from the fetch response.

Reply:
No there's no need, we already call it to set _templateVersionNumber on line 494

Reply:
👍